extensions/blocks/eventbrite/save.js

```diff
@@ -55,7 +49,9 @@ function saveButton( attributes ) {
 			<RichText.Content
 				className={ buttonClasses }
 				href={ url }
-				id={ createWidgetId( eventId ) }
+				// Placeholder id, preg replaced with a unique id generated in PHP when the block is rendered.
+				// IMPORTANT: do not remove or change unless you also update the render function in eventbrite.php.
+				id="eventbrite-placeholder-id"
```

Comment:
I suppose the only problem here is that if the server render function doesn't run you could potentially end up with two elements with the same ID. We could get around that by appending a random number and updating the regex in the PHP. 

It's a super unlikely edge case though, which most browsers would cope with, so it might not be worth bothering with.

Reply:
That's a great point--I'll make an update to address that.

Reply:
✅

Reply:
However, it can't be a random number, because then the block validation fails when the already existing block is loaded in the editor. So I'm just using the `eventId` to help mitigate this.